plugins/ci/pkg/ci/ci.go

```diff
@@ -345,6 +347,16 @@ func getGitInfo(repoName string) (gitInfo, error) {
 		return info, err
 	}
 
+	var commitMessage string
+	commitMessage, err = GetResultsFromCommand("git", "log", "--pretty=format:%s", "-1")
+	if err != nil {
+		logrus.Warn("Unable to get GIT Commit message")
+		return info, err
```

Comment:
The strategy for `getInfo()` is all or nothing right? . either you get all the info or nothing.

Reply:
Yup, it's all or nothing.